main.go

```diff
@@ -22,8 +26,18 @@ const (
 	envVarOrchestrionLogFile  = "ORCHESTRION_LOG_FILE"
 	envVarOrchestrionLogLevel = "ORCHESTRION_LOG_LEVEL"
 	envVarToolexecImportPath  = "TOOLEXEC_IMPORTPATH"
+	envVarCPUProfiling        = "CPU_PROFILING"
+	envVarHeapProfiling       = "HEAP_PROFILING"
+	envVarExecutionTracing    = "EXECUTION_TRACING"
+	envVarProfilePrefix       = "PROFILE_PREFIX"
```

Comment:
Environment variables that are CLI flag stand-in should be named `ORCHESTRION_<SHOUT_CASE_OF_FLAG_NAME>`, so the relationship is clear.

Reply:
Done